### PR TITLE
Use CAPWAP-AC with HTTP-API

### PR DESCRIPTION
**Pull request by mgumz**
CAPWAP-AC got an upgrade which provides a simple HTTP API to take over
the functionality of the now not-existent CLI.

Note: The support library *jsx* (a JSON Encoder/Decoder) needs special
treatment in rebar.config due to subtle behaviour of rebar3 which
deletes the result of certain compile time checks of *jsx* resulting
in *jsx* being not able to encode / decode Erlang maps. See
https://github.com/erlang/rebar3/issues/1360#issuecomment-256929014
for more details.

### CODE REVIEW

File: rebar.config [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-
-        {sys_config, "./config/sys.config"},
-        {vm_args, "./config/vm.args"},
-
-        {dev_mode, true},
-        {include_erts, false},
-
-        {extended_start_script, true}]
-}.
+{relx, [
```
Comment (RoadRunnr) [inline rebar.config]: I don't like the whitespace only changes

File: rebar.config [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 {deps, [
-	{capwap, ".*", {git, "git://github.com/travelping/capwap", {tag, "1.9.1"}}},
-	{gen_listener_tcp, ".*", {git, "git://github.com/travelping/gen_listener_tcp", "master"}},
-	{netdata, ".*", {git, "git://github.com/RoadRunnr/erl_netdata", "master"}}
+    {capwap,           {git, "git://github.com/travelping/capwap",
+                           {branch, "master"}}},
+    {gen_listener_tcp, {git, "git://github.com/travelping/gen_listener_tcp",
+                           {branch, "master"}}},
+    {netdata,          {git, "git://github.com/RoadRunnr/erl_netdata",
+                           {branch, "master"}}}
```
Comment (RoadRunnr) [inline rebar.config]: updateing the deps style is ok, but avaid the line break and don't tabulate the entries

File: rebar.config (hunk 1 above)
Comment (mgumz) [reply rebar.config]: adjusted

File: rebar.config (hunk 2 above)
Comment (mgumz) [reply rebar.config]: adjusted

File: rebar.config (hunk 1 above)
Comment (mgumz) [reply rebar.config]: the whitespace in that file is not really in one style or the other...

**Comment by mgumz** [comment]
out of date

**Pull request closed without merging**